hack/crds/clusterimagesets.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (devel)
+  labels:
+    operators.coreos.com/hive-operator.openshift-operators: ""
+  name: clusterimagesets.hive.openshift.io
+spec:
+  conversion:
+    strategy: None
+  group: hive.openshift.io
+  names:
+    kind: ClusterImageSet
+    listKind: ClusterImageSetList
+    plural: clusterimagesets
+    shortNames:
+    - imgset
+    singular: clusterimageset
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.releaseImage
+      name: Release
+      type: string
+    name: v1
+    schema:
+      openAPIV3Schema:
+        description: ClusterImageSet is the Schema for the clusterimagesets API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
```

Comment:
nit: representing...represents in the same sentence. Maybe worth rephrasing. 
Also, while we try to break the lines 32 and 33, line 34 is a very long one.  Just raising it to your awareness. If you still wanna go ahead with it - ok with me.

Reply:
This is not actually something we own. This is taken directly from hive's CRD. Plus the `kind` description is very likely defined in kubernetes core code.